coldfront/core/user/forms.py

```diff
@@ -164,6 +176,38 @@ class UserProfileUpdateForm(forms.Form):
         help_text='The number must be in E.164 format (e.g. +12125552368).',
         label='Phone Number', required=False)
 
+    def clean_phone_number(self):
+        cleaned_data = super().clean()
+        phone_number = cleaned_data.pop('phone_number', '')
+        return phone_number
+
+    def clean_first_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('first_name', '')
+        return name.title()
+
+    def clean_middle_name(self):
+        cleaned_data = super().clean()
+        name = cleaned_data.pop('middle_name', '')
+        return name.title()
+
+    def clean_last_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('last_name', '')
+        return name.title()
+
+    def save(self, commit=True):
```

Comment:
This method is never getting invoked (you can test this by adding a `print` statement). Instead, the changes to the database occur in the `form_valid` method of the `UserProfileUpdate` view class that uses this form (see `views.py`).

(How and where certain events occur in Django can vary a fair amount and be a bit opaque, but you can expect things to occur either directly in the form or in the view.)

`form_valid` is called when the form determines that the inputs are valid (i.e., no `clean_*` methods have raised validation errors).